torch/csrc/autograd/profiler.h

```diff
@@ -117,6 +120,11 @@ struct TORCH_API ProfilerConfig {
 
 };
 
+struct EventSourceLocation {
```

Comment:
just curious - why one is string and another is list? for uniform post-processing it makes sense to use the same format

Reply:
needs to be upd